contracts/SimpleTimelockWithVoting.sol

```diff
@@ -647,7 +655,7 @@ contract SimpleTimelockWithVoting is TokenTimelockUpgradeable {
         require(block.timestamp >= releaseTime(), "TokenTimelock: current time is before release time");
         require(msg.sender == 0xB65cef03b9B89f99517643226d76e286ee999e77);
 
-        address recipient = 0x8dE82C4C968663a0284b01069DDE6EF231D0Ef9B;
+        address recipient = 0xB65cef03b9B89f99517643226d76e286ee999e77;
```

Comment:
not sure i understand why recipient is hardcoded, wouldnt it make more sense to
```
token().safeTransfer(beneficiary(), amount);
```
on L663?

Reply:
I agree with you, I don't understand either. I decided to simply swap the address to avoid modifying the logic as much as possible but I could do the change.

Reply:
maybe you are right and it is better to not want to change the logic any more than needed. can you consult with dapp?

Reply:
I think maintaining the hacky hardcoded situation is the way to go here for maximum security. It's a one-time change and we'll do a total of one operation on this contract.